### PR TITLE
Coviu Demo Plugin

### PR DESCRIPTION
**Pull request by sureshponraj90**

### CODE REVIEW

**Comment by jayenashar** [comment]
hi @sureshponraj90 is this a template we can use?  or is just meant for demos?  why webpack 3 and not webpack 5?

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+var gulp = require("gulp");
```
Comment (jerry-coviu) [inline gulpfile.js]: don't put building process file into github

Comment (jayenashar) [reply gulpfile.js]: does it need a BFG Repo-Cleaner so it's not in the history?

**Comment by jerry-coviu** [comment]
@sureshponraj90 please decline this PR, create new one without gulpfile

**Pull request closed without merging**